record_imu_cam/plugin.cpp

```diff
@@ -90,12 +85,15 @@ class record_imu_cam : public plugin {
 	std::ofstream cam1_wt_file;
 	const std::shared_ptr<switchboard> sb;
 
+	boost::filesystem::path record_data; 
```

Comment:
Mark these `const`, and compute them with braces (like we do `sb`), in the order they are initialized here.

Reply:
I see that you approved the PR. Would you like the above addressed or no?

Reply:
Please address before merging; all else looks good.